### PR TITLE
Update service-connection-tool-not-download-updates.md

### PR DESCRIPTION
**Pull request by rashid2982**
Updated the workaround and added Access requirements

### CODE REVIEW

**Comment by PRMerger20** [comment]
@rashid2982 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: support/mem/configmgr/service-connection-tool-not-download-updates.md [hunk 1]
```diff
@@ -31,6 +31,12 @@ This issue occurs in one of the following situations:
 - The specific URLs that are required by the service connection point aren't included in the allowlist on your firewall or proxy server.
 
 ## Resolution
+- If you don't have already, Please consider upgrading the server to Server 2016 or above, 
+- Please make sure you have all the internet rules and exceptions in place for your proxy/firewall to allow the traffic, 
+- Related docs
+[Internet Access Requirement](https://docs.microsoft.com/en-us/mem/configmgr/core/plan-design/network/internet-endpoints). 
+[ConfigMgr Proxy Exceptions](https://docs.microsoft.com/en-us/archive/blogs/configmgrdogs/configuration-manager-proxy-exceptions)
+
```
Comment (rashid2982) [inline support/mem/configmgr/service-connection-tool-not-download-updates.md:39]: Need to add this too

Please make sure below 4 ciphers are enabled on the server with Service Connection Point role
- TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384 (0xc030)
- TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256 (0xc02f)
- TLS_DHE_RSA_WITH_AES_256_GCM_SHA384 (0x9f)  
- TLS_DHE_RSA_WITH_AES_128_GCM_SHA256 (0x9e)  
[Check Ciphers](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/operations/manage-ssl-protocols-in-ad-fs#enabling-or-disabling-additional-cipher-suites)

**Comment by PRMerger5** [comment]
@helenclu : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger18** [comment]
@helenclu : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**